Lib/gftools/builder/__init__.py

```diff
@@ -455,10 +455,7 @@ def build_a_static_format(self, format, directory, postprocessor):
                 "output_dir": directory,
                 "optimize_cff": CFFOptimization.SUBROUTINIZE,
             }
-            if source.endswith("ufo"):
-                if "interpolate" in args:
-                    del args["interpolate"]
-            else:
+            if source.endswith("glyphs"):
                 args["interpolate"] = True
```

Comment:
Do we need to add the `-i` flag on .designspace sources? (I don't know.)

Reply:
yes, if you want the instances, otherwise you get only the masters.

Reply:
I think the previous logic is ok. If it's a ufo, it'll delete the interpolate key if it exists, otherwise it'll set it for a ds or glyphs file. You ok for me to rollback my last commit and leave this alone?

Reply:
Right, I'm being a clown here. Just realised we're constructing the args a fresh in the loop just above. Ofc I'll just drop it for ufos.

Reply:
this should be good.